utils/libnetwork/Makefile

```diff
@@ -21,6 +21,7 @@ PKG_MAINTAINER:=Gerard Ryan <G.M0N3Y.2503@gmail.com>
 
 PKG_BUILD_DEPENDS:=golang/host
 PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
```

Comment:
If you're not compiling for MIPS, this is pointless.

Reply:
Removed